### PR TITLE
Add other groups to the readme

### PR DESCRIPTION
**Pull request by ire4ever1190**
This adds our group along with last semesters group to the readme

I put the names in a drop down so (hopefully) as more groups use this repo and append to it, it won't take up the entire readme

### CODE REVIEW

**Comment by ire4ever1190** [comment]
I just used the sorting I used for the meeting minutes, think I got it from some random website.

Where do you think the mysterious Lachlan Van Der Klift guys name should be placed?

**Comment by LvandoApps** [comment]
just not last imo, i heard bad things happen at 3am to repositories that do that

**Comment by ire4ever1190** [comment]
Zero clue where else to put them :thinking: is there a place lower than bottom?

Just add a commit to change them or write suggestion with a review

File: README.md [hunk 1]
```diff
@@ -8,14 +8,36 @@ Automatarium is a modern take on the useful [JFLAP](https://www.jflap.org/), but
...
+</details>
+
+<details>
+<summary>Group 3</summary>
+
+- [Ope Abbas](https://github.com/OpeAbbas)
+- [Sidhra Fernando-Plant](https://github.com/SidhraFernando-Plant)
+- [Jake Leahy](https://github.com/ire4ever1190)
+- [Aung Pyae Sone](https://github.com/eddie7788)
+- [Lachlan Van Der Klift](https://github.com/LvandoApps)
```
Comment (LvandoApps) [inline README.md:38]:
```suggestion

```

Comment (LvandoApps) [inline README.md:38]:
```suggestion
- [Ope Abbas](https://github.com/OpeAbbas)
- [Sidhra Fernando-Plant](https://github.com/SidhraFernando-Plant)
- [Lachlan Van Der Klift](https://github.com/LvandoApps)
- [Jake Leahy](https://github.com/ire4ever1190)
- [Aung Pyae Sone](https://github.com/eddie7788)
```

**Review by LvandoApps — APPROVED** [review]
my suggestions

**Pull request merged**